src/service/command/meme/takopi.ts

```diff
@@ -1,15 +1,19 @@
 import type { MemeTemplate } from '../../../model/meme-template.js';
 
-const takopiFlags = ['f'] as const;
+const takopiFlags = ['f', 'c'] as const;
```

Comment:
```suggestion
const takopiFlags = ['f'] as const;
const takopiOptions = ['c'] as const;
```
フラグではなくオプションを使ったほうがよいかと思います